MANIFEST.in

```diff
@@ -2,3 +2,4 @@ include versioneer.py
 include nvflare/_version.py
 include nvflare/libs/*.so
 include nvflare/fuel/utils/*.json
+include nvflare/private/fed/app/simulator/log_config.json
```

Comment:
just happen to see this now, why do we need this file?
@SYangster

Reply:
Ah good catch, this file was moved to `nvflare/fuel/utils/log_config.json`. I can remove this from the MANIFEST.in